src/pages/library/dataBrowser-utils.js

```diff
@@ -39,6 +40,14 @@ export const datasetReleasePolicies = {
   releasepolicy_other: { policy: 'SnapshotReleasePolicy_Other', label: 'Other', desc: 'Misc release policies' }
 }
 
+export const isWorkspace = dataset => {
+  return _.toLower(dataset['dcat:accessURL']).includes('/#workspaces/')
+}
```

Comment:
FYI that the above could be written as below for concision and a bit better error handling (thanks to Lodash):

```javascript
export const isWorkspace = _.flow( 
  _.toLower,
  _.includes('/#workspaces/')
)
```

Similarly for the function below.